.travis.yml

```diff
@@ -15,12 +15,39 @@ addons:
 cache:
   directories:
     - $HOME/.cabal
-ghc:
-  - '8.10'
-  - '8.8'
-  - '8.6'
+    - $HOME/.stack
+    - $TRAVIS_BUILD_DIR/.stack-work
+
+matrix:
+  include:
+
+  # Cabal
+  - ghc: '8.10'
+  - ghc: '8.8'
+  - ghc: '8.6'
+
+  # Stack
+  - ghc: '8.10.4'
+    env: STACK_YAML="$TRAVIS_BUILD_DIR/stack.yaml"
```

Comment:
Why is the `stack.yaml` in `$TRAVIS_BUILD_DIR`? Shouldn't it be in the current directory?

Reply:
fixed.